quick_start/sample_app.py

```diff
@@ -0,0 +1,382 @@
+import enum
+import json
+import uuid
+from typing import ClassVar
+import inflection as inflection
```

Comment:
This package (and tqdm) doesn't come with python by default, so users will get an error.
I think the simplest thing to do here is to include a requirements.txt file and ask for users to install it